### PR TITLE
feat: ui only for migrate (review, signing, congrats)

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Related #7552
- [x] UI only: #7561
- [x] UI only: #7562 (small portion)
- [x] UI only: #7563

### Preview
- https://deploy-preview-7732--koda-canary.netlify.app/migrate/review?collectionId=12345789&source=ksm&destination=ahp
- https://deploy-preview-7732--koda-canary.netlify.app/migrate/sign?collectionId=12345789&source=ksm&destination=ahp
- https://deploy-preview-7732--koda-canary.netlify.app/migrate/congrats?collectionId=12345789&source=ksm&destination=ahp

### <samp>🤖 Generated by Copilot at bef6cb2</samp>

Added a new feature to migrate collections between networks. Created a `useMigrate` function to manage the migration state and logic. Implemented four page components (`congrats.vue`, `review.vue`, `sign.vue`, and `index.vue`) and one content component (`MigrateContent.vue`) to handle the UI and navigation of the migration process. Refactored the `MigrateHeader.vue` component to use the `useMigrate` function. Removed the footer from the migrate page.

### <samp>🤖 Generated by Copilot at bef6cb2</samp>

> _Sing, O Muse, of the mighty `useMigrate` function_
> _That the skillful coder devised for the migration process_
> _To define and watch the source and destination networks_
> _And to share the state with all the components of the page_

### CODE REVIEW

File: components/migrate/MigrateReviewSign/MigrateReview.vue [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+      <div class="">{{ $t('mint.nft.modal.totalFee') }}:</div>
+      <div class="is-flex is-align-items-center">
+        <div class="has-text-k-grey is-size-7 mr-2">$108</div>
+        <div>10.9 KSM</div>
+      </div>
+    </div>
+
+    <NeoField>
+      <NeoCheckbox v-model="agree">
+        I understand this action is irreversible once done.
```
Comment (vikiival) [inline components/migrate/MigrateReviewSign/MigrateReview.vue:154]: Please use translations

File: components/migrate/MigrateReviewSign.vue [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  section: 'review' | 'sign'
+}>()
+
+const promptModal = async () => {
+  const instance = neoModal.open({
+    component: MigrateModal,
+    contentClass: 'k-shadow border',
+    trapFocus: true,
+  })
+  const result: { action: 'yes' | 'no' } = await instance.promise
```
Comment (daiagi) [inline components/migrate/MigrateReviewSign.vue:56]: how about a boolean instead?

**Review by daiagi — COMMENTED** [review]
Beautiful work. 

Som eTablet size issues:

image


image


image


image

**Comment by preschian** [comment]
@vikiival @daiagi updated

**Comment by daiagi** [comment]
I think it should transition to  mobile mode earlier (wider screen)

### 855px:

image


###711px:
image

**Review by daiagi — APPROVED** [review]

**Comment by prury** [comment]
dark mode:
image

**Comment by exezbcz** [comment]
Amazing sir! 

some things

### top section 
image

- when the section (tab) is not active, please make it regular font weight.
- design: 
image

- I have added the same styling as in admin panel
image

- basically: when you are on the first step, next steps are grey, when you complete step and proceed, the step turns black 
- each active state is bold and black



### route
image

- please remove hover
- change stroke to k-shade




### cta 
image

- when the button is checked, it should change its button label to proceed to signing 
image
- what we have now is the disabled button waiting for you to check the checkbox. There will also be a few states, like insufficient funds etc.

### final graphics 
- I have added dark mode to Figma. Could you please change it 
image



There will be some changes, mainly I think, to the signing part, and one addition to the congrats screen. I will create issue for that.

thank you

**Comment by yangwao** [comment]
Thanks!
pay 100 usd

**Comment by yangwao** [comment]
Thanks!
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 4.23 USD/DOT ~ 23.641 $DOT
🧗 [1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p](https://polkadot.subscan.io/account/1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)
🔗 [0x791016fbb27d12306e46f41cbf4c7caaf80f0aca443cebcf1689fa3adbaf7016](https://polkadot.subscan.io/extrinsic/0x791016fbb27d12306e46f41cbf4c7caaf80f0aca443cebcf1689fa3adbaf7016)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**